service/httpCall.go

```diff
@@ -79,7 +80,7 @@ func MakeHTTPCall(scenarios []model.TestScenario) (map[string]bool, int) {
 	for _, test := range testScenarios {
 		result := make(map[string]interface{})
 		if test.Body != nil {
-			switch test.Type {
+			switch strings.ToLower(test.Type) {
```

Comment:
I think it's better to use ```strings.Equalfold()```.

Reply:
I think EqualFold() is not applicable here as switch case is on test.Type. EqualFold returns bool.